biojava-structure-gui/src/test/java/org/biojava/nbio/structure/gui/TestSimilarityCalc.java

```diff
@@ -0,0 +1,98 @@
+package org.biojava.nbio.structure.gui;
+
+import junit.framework.TestCase;
+import org.biojava.nbio.structure.Atom;
+import org.biojava.nbio.structure.Structure;
+import org.biojava.nbio.structure.StructureTools;
+import org.biojava.nbio.structure.align.StructureAlignment;
+import org.biojava.nbio.structure.align.StructureAlignmentFactory;
+import org.biojava.nbio.structure.align.model.AFPChain;
+import org.biojava.nbio.structure.align.model.AfpChainWriter;
+import org.biojava.nbio.structure.align.seq.SmithWaterman3DParameters;
+import org.biojava.nbio.structure.align.seq.SmithWaterman3Daligner;
+import org.biojava.nbio.structure.align.util.AtomCache;
+import org.junit.Test;
+
+/**
+ * Created by andreas on 8/3/16.
+ */
+public class TestSimilarityCalc extends TestCase{
```

Comment:
No need to extend TestCase, the @Test annotation in the individual methods is enough